app/policies/application_policy.rb

```diff
@@ -0,0 +1,55 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    scope.where(:id => record.id).exists?
+  end
+
+  def create?
+    #false #false is default
+    user.present?
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    #false #false is default
```

Comment:
You would want to remove these comments if committing in real life.

Reply:
I certainly would never leave this kind of comment in real development. this is just for my learning purposes.